codegen/idris-codegen-c/Main.hs

```diff
@@ -16,17 +16,19 @@ import Paths_idris
 
 import Util.System
 
-data Opts = Opts { inputs :: [FilePath],
+data Opts = Opts { really :: Bool,
+                   inputs :: [FilePath],
                    interface :: Bool,
                    output :: FilePath }
 
-showUsage = do putStrLn "Usage: idris-c <ibc-files> [-o <output-file>]"
+showUsage = do putStrLn "Usage: idris-codegen-c --yes-really <ibc-files> [-o <output-file>]"
```

Comment:
Perhaps the usage message should state that the idea is that this gets called by Idris rather than users?